apps/site/src/components/page-sidebar.tsx

```diff
@@ -61,7 +62,7 @@ type SidebarPageSectionProps = {
 const highlightSection = (isSectionSelected: boolean) => ({
   borderLeft: `3px solid ${
     isSectionSelected ? themeImport.palette.purple[700] : "transparent"
-  }`,
+  } !important`,
```

Comment:
Any way we can easily achieve this using specificity where this is used? I think it's best practice to reserve `!important` to a last resort, so just wanted to check.

Reply:
I agree, that `!important` is used to override MUI applying `border:none` when it's `:focus-visible`. So I guess it's fair to use here to prevent MUI removing the indicator border.

Reply:
But can't we use the `:focus-visible` selector to prevent having to use `!important` here then? If it's not worth the trouble, then I'm okay with us leaving it. Just think it's worth trying to avoid `!important` in the codebase where we can.

Reply:
Hmm, yes that makes sense. Thanks for reminding me this angle 😊 

Updated it here https://github.com/blockprotocol/blockprotocol/pull/1225/commits/34265d03cc9a491e6813f4b75982c1339e079538